packages/editor/src/lib/editor/managers/WatermarkManager.ts

```diff
@@ -117,6 +117,7 @@ export class WatermarkManager {
 			// we still fallback to the local watermark.
 			this.forceLocal = true
 			this.setWatermarkSrc(watermark)
+			watermark.onerror = null
```

Comment:
Without this we could end up in a cycle: 
1. We couldn't load the cd version.
2. Error handler gets called and changes the src.
3. If this one also failed we end up in a cycle of error handler getting called -> we replace the src -> it tries to refetch the image -> error again. 

Think it's safer to just do it once.

Reply:
![image](https://github.com/tldraw/tldraw/assets/2523721/80ab49ce-e136-4add-a640-75badf51b0fc)

Reply:
ah good call!